src/components/Containers/TopSection/TopSection.jsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import styled from "styled-components";
+
+import SideGraphic from "../../../assets/top-section-graphic.svg";
+import { media } from "../../../utils";
+import { Text } from "../../Texts";
+import { Button } from "../../Inputs";
+
+export const TopSection = ({ ...props }) => (
+  <MainContainer {...props}>
+    <MobileGraphic>
+      <Graphic src={SideGraphic} alt="Graphic" />
+    </MobileGraphic>
+    <FirstColumn>
+      <JosefinTitle size="xxlarge" bold="true">
+        Prospective Medical Professionals
+      </JosefinTitle>
+      <SText size="default">
+        A registered charity dedicated to connecting aspiring students to health
+        sciences opportunities
+      </SText>
+      <SButton>
+        <ButtonText size="default" bold="true">
+          DONATE
+        </ButtonText>
+      </SButton>
+    </FirstColumn>
+    <SecondColumn>
+      <Graphic src={SideGraphic} alt="Graphic" />
+    </SecondColumn>
+  </MainContainer>
+);
+
+const MainContainer = styled.div`
+  display: flex;
+  margin: 5em;
+  ${media(
+    "mobile",
+    `
+        margin: 1em;
+        flex-direction: column;
+        `
+  )};
+`;
+
+const SText = styled(Text)`
+  ${({ theme }) => `
+    font-family: ${theme.font.lato};
+    color: ${theme.colors.navy};
+    margin-top: 0;
+  `};
+  ${media(
+    "mobile",
+    `
+        line-height: 1.3;
+        `
+  )};
+`;
+
+const JosefinTitle = styled(SText)`
+  ${({ theme }) => `
+    font-family: ${theme.font.josefin};
+    margin: 1em 0 0 0;
+  `};
+  ${media(
+    "mobile",
+    `
+        font-size: 2.5rem;
+        `
+  )};
+`;
+
+const ButtonText = styled(SText)`
+  margin: 0 0 0 0;
+`;
+
+const Graphic = styled.img`
+  width: 100%;
+  height: auto;
+`;
+
+const SButton = styled(Button)``;
```

Comment:
If there's no styling, you can directly add it as "Button", there's no need to style it